cookbook/persistance/key-value.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: page
+title: "Storing key-value data on disk"
+permalink: /cookbook/persistence/key-value/
+---
+
+If you have a relatively small collection of key-values that you'd like to save, you should use the [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) APIs
+
+Normally we would have to write native platform integration for storing data for both platforms, but fortunetly [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin can be used for that. 
+
+Shared preferences plugin wraps `NSUserDefaults` (on iOS) and `SharedPreferences` (on Android), providing a persistent store for simple data. Data is persisted to disk automatically and asynchronously.
+
+## Setup
+
+Before you start, you need to add [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin to you `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  shared_preferences: "<newest version>"
+```
+
+## Saving, reading and removing data
```

Comment:
Maybe we could create 3 sections here::

  1. Saving Data
  2. Reading Data
  3. Removing Data

In the example below, the order is Read (null), Set, Delete, which might be a bit funny for an introduction.